$MirOS: ports/lang/nwcc/patches/patch-test_sh,v 1.1 2008/10/30 17:11:16 tg Exp $

	Add MirBSD; honour $CC, $CFLAGS

--- test.sh.orig	Wed Mar  4 16:18:03 2009
+++ test.sh	Fri Apr 17 14:26:27 2009
@@ -57,7 +57,7 @@ SYS=`uname -s`
 MACH=`uname -m`
 
 if test "$SYS" = "Linux" || test "$SYS" = "FreeBSD" || \
-	test "$SYS" = "OpenBSD"; then
+	test "$SYS" = "OpenBSD" || test "$SYS" = "MirBSD"; then
 	if test "$MACH" = "x86_64" || test "$MACH" = "amd64"; then
 		if test "$NWCC_ASM" != ""; then
 			BASEASM=`basename $NWCC_ASM`
@@ -126,8 +126,9 @@ if test `uname -p` = sparc; then
 	GCC_CFLAGS="-m64"
 fi
 
+: ${CC=gcc}
 try_files() {
-	if ! gcc $GCC_CFLAGS $FILES >/dev/null 2>/dev/null; then
+	if ! $CC $CFLAGS $GCC_CFLAGS $FILES >/dev/null 2>/dev/null; then
 		echo "gcc error"
 		continue
 	fi	
